sonic-net/sonic-buildimage
# [Netberg][Barefoot] Added support for Aurora 710

#### Why I did it

Support Intel Tofino based platforms Netberg Aurora 710
ASIC:  Intel Tofino BFN-T10-032D
Pors: 32x 100G

#### How I did it

Added specification to device/netberg directory
Added platform/barefoot/sonic-platform-modules-netberg contains kernel modules, scripts and sonic_platform packages.
Modified the platform/barefoot/platform-modules-netberg.mk to include Aurora 710 related ID.

#### How to verify it

Build SONiC

Install the image on the device and verify the related components are installed and shown correctly.

Verification output - [nba710-showplatform-master-010822.txt](https://github.com/sonic-net/sonic-buildimage/files/9478127/nba710-showplatform-master-010822.txt)

#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog




#### Link to config_db schema for YANG module changes


#### A picture of a cute animal (not mandatory but encouraged)
